docs/OESDLBugBar.md

```diff
@@ -0,0 +1,86 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
+
+This bug bar and the ratings are derived based on [Microsoft SDL Bug bar](https://aka.ms/sdlbugbar) to apply for the enclaves, specifically to the enclaves created using Intel SGX technology. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. 
+
+The bug bar describes different severities for the bugs affecting enclaves. Enclaves are the privileged execution environment which is a security domain. There is a security boundary between enclave and rest of the system, this is a stronger security boundary that should be defended. It is important to understand the enclave security boundary, it should be apt to consider this as a confidentiality & integrity boundary. Anything that is inside enclave should never be accessible or modified from outside of the enclave. 
```

Comment:
Grammar: "boundary, it" should be in two different sentences.

 Also please split source (markdown) lines into shorter lines to make comments/diffs easier to review, rather than having one source line span 6 sentences that wrap across 5 browser window lines.